static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.js

```diff
@@ -6,11 +6,24 @@ import { dfpEnv } from 'commercial/modules/dfp/dfp-env';
 import once from 'lodash/once';
 import { prebid } from 'commercial/modules/prebid/prebid';
 
+const isGoogleWebPreview: () => boolean = () => {
+    try {
+        return navigator.userAgent.indexOf('Google Web Preview') > -1;
+    } catch (exception) {
+        return false;
+    }
+};
+
+if (!isGoogleWebPreview()) {
+    require('prebid.js/build/dist/prebid'); // eslint-disable-line global-require
+}
```

Comment:
When does this get called?

Reply:
It gets called on the import of `prepare-prebid`; which is a compile time import in webpack.